Swift Code Challenge/Views/Components/CircleImageComponent.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  CircleImageComponent.swift
+//  Swift Code Challenge
+//
+//  Created by Luiz Eduardo Mello dos Reis on 30/06/22.
+//
+
+import SwiftUI
+
+struct CircleImage: View {
+    var userImage: String
+    var body: some View {
+        AsyncImage(url: URL(string: userImage))
+            .aspectRatio(contentMode: .fit)
+            .frame(width: 128, height: 128)
```

Comment:
To make this reusable, we could remove this so the view that implements it could specify it.